PyRDF/__init__.py

```diff
@@ -19,4 +19,25 @@ def use(backend_name, conf = {}):
     elif backend_name == "spark":
         current_backend = Spark(conf)
     else:
-        raise Exception(" Incorrect backend environment \"{}\"".format(backend))
\ No newline at end of file
+        raise Exception(" Incorrect backend environment \"{}\"".format(backend))
+
+def include(includes_list=[]):
```

Comment:
Proposal:
- `includes_list` is renamed to `includes` and does not have a default value  
- We have a module-level variable here called `includes` (defined right after `current_backend`) which is initialized to `[]`.
- We need to support the following sequences of operations:
  - include + use
  - use + include + use
  - include + include
- To support those sequences of operations, we need that `include` appends to the module-level list the new included headers, and that the backends read that module-level list.

Reply:
Yup @etejedor . I agree. That makes sense. I'll make these changes  👍

Reply:
Can I retain the name of `includes_list` @etejedor ? There'd be a clash in the names otherwise

Reply:
Ok!

Reply:
One question @etejedor . If I declare an `includes` variable in PyRDF module level, then, how would I get the value of that variable inside a Backend class ? Is `from .. import includes` fine ?

Reply:
Yes, or `from PyRDF import includes` should work too. Doesn't `Proxy` import `current_backend` already? Same thing here.

Reply:
Yup. Inside `Proxy` I'm doing a relative import. So, I have to do something similar inside `Backend` as well